dataprofiler/labelers/data_processing.py

```diff
@@ -1555,8 +1555,8 @@ def __init__(
             random_state = random.Random(random_state)
         elif isinstance(random_state, (list, tuple)) and len(random_state) == 3:
             # tuple required for random state to be set, lists do not work
-            if isinstance(random_state[1], list):
-                random_state[1] = tuple(random_state[1])  # type: ignore
+            if isinstance(random_state[1], list) and isinstance(random_state, list):
```

Comment:
should we first check if `random_state` is a list and then check if `random_state[1]` is a list? Otherwise wouldn't the first boolean condition error if `random_state` isn't a list to begin with?
```suggestion
            if isinstance(random_state, list) and isinstance(random_state[1], list):
```

Reply:
It technically doesn't matter here since we've already checked that `random_state` is either a `list` or a `tuple` in line 1556, which means `random_state` is indexable. But I agree it's better practice to check `random_state` before `random_state[1]` in case the code changes later/for readability